src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -219,6 +219,12 @@ private static void WriteBoundAttribute(JsonWriter writer, BoundAttributeDescrip
                 writer.WriteValue(boundAttribute.IsEnum);
             }
 
+            if (boundAttribute.IsEditorRequired)
```

Comment:
Does the doc comment for `IsEditorRequired` explain what it does? Cause I think it's a bit ambiguous just from the name.

Reply:
It does not as yet - I can add some. Here's some background about what this is about - https://github.com/dotnet/aspnetcore/issues/11815#issuecomment-679290378